pkg/system/installation.go

```diff
@@ -440,7 +440,7 @@ func dbxrootInstallToDisk(disk string, t dogeboxd.Dogeboxd) error {
 }
 
 func dbxrootInstallToT6(disk string, t dogeboxd.Dogeboxd) error {
```

Comment:
We no longer need this function if it's basically the exact same with a different variant